kubetest/azure.go

```diff
@@ -231,6 +235,14 @@ func (c *Cluster) generateTemplate() error {
 				ClientID: c.credentials.ClientID,
 				Secret:   c.credentials.ClientSecret,
 			},
+			ExtensionProfiles: []map[string]string{
```

Comment:
can someone from azure review this commit? I've no idea what these URLs or workflow should look like. alternatively I'd look at some docs for this if someone has a link :-)

Reply:
Added comments for this in the next commit.